### DISCUSSION TITLE
How to Keep Blog Sidebar Scroll Position After Selecting a Post?

### DISCUSSION THREAD

**Discussion by ruanqizhen** [facebook/docusaurus#10793] [Q&A]
I created a blog with many posts and configured the sidebar using `blogSidebarCount: 'ALL'` to list all posts. However, I encountered an issue: when I scroll down the sidebar to select an older post and click it, the sidebar automatically scrolls back to the top after the selected post loads. Is there a setting to keep the sidebar's scroll position intact, similar to how the doc sidebar behaves?

**Comment by slorber** [comment]
Unfortunately not: the blog layout unmounts and remounts when navigating from blog list view to blog detail view. It's not ideal but we don't have any option currently to achieve that, this requires an internal blog plugin refactoring.

Also related to https://github.com/facebook/docusaurus/issues/2891
